src/OneDrive.Sdk.Authentication.Xamarin.Android/ManagedProtection.cs

```diff
@@ -0,0 +1,310 @@
+//
```

Comment:
I agree, it is included in the referenced Mono package, System.Security.dll, but is internal as it is called by the [ProtectedData](https://github.com/mono/mono/blob/0bcbe39b148bb498742fc68416f8293ccd350fb6/mcs/class/System.Security/System.Security.Cryptography/ProtectedData.cs) class when DPAPI is unavailable (e.g. non-Windows).  However, it looks like it may get omitted due to a conditional compilation for mobile.  I'll see if I can make a separate pull request for mono to get it included, but thought it would best to include it directly in the mean time.  If it gets exposed in Mono, the class can be omitted completely and the caller changed to use ProtectedData instead.